trade-finance-manager-api/api-tests/types/test-api.ts

```diff
@@ -0,0 +1,59 @@
+import { AnyObject } from '@ukef/dtfs2-common';
+import { Response } from 'supertest';
+import { IncomingHttpHeaders } from 'http';
+import { TestUser } from './test-user';
+import { File } from './file';
+
+export type TestRequestWithoutHeaders = (data: AnyObject) => {
+  to: (url: string) => Promise<Response>;
+};
+
+type TestGetWithHeaders = (
+  url: string,
+  {
+    headers,
+    query,
+  }?: {
+    headers: IncomingHttpHeaders | undefined;
+    query: AnyObject | undefined;
```

Comment:
```suggestion
    headers?: IncomingHttpHeaders;
    query?: AnyObject
```

Reply:
Thanks @MarRobSoftwire 

Not for this PR but it would be nice to some day look into improving these, seeing a lot of instances of `| undefined`, I don't understand why we should explicitly need to define something as "or undefined" 😢